d2bs/kolbot/libs/OOG.js

```diff
@@ -1016,6 +1016,23 @@ MainLoop:
 		return position;
 	},
 
+	getQueueTime: function() {
+		const control = getControl(4, 427, 234, 300, 100);
+
+		if (control && control.getText()) {
+			const text = control.getText(); //You are in line to create a game.,Try joining a game to avoid waiting.,,Your position in line is: ÿc02912
+			if (text.indexOf(getLocaleString(11026)) !== -1) {
+				const result = /ÿc0(\d*)/gm.exec(text);
+				if (result && typeof result[1] === 'string') {
+
```

Comment:
random newline

Reply:
[eslint](https://gist.githubusercontent.com/Nishimura-Katsuo/2d6866666c7acf10047c486a15a7fe60/raw/99ef9c2995929c492ef856772ff346e0f19709cd/.eslintrc.js)

`	"padding-line-between-statements": ["error",
			{"blankLine": "always", "prev": "*", "next": ["return", "block-like"]},`

I dont like it either, but its your standard

Reply:
I think nishi wrote the eslint, but nonetheless, feel free to submit change to the eslint as well